### ISSUE TITLE
Prompt Limits on Develop

### ISSUE THREAD

**Issue by ScuffedNewt** [lk-arpg/lorekeeper#1103]
I think adding Junijwi's prompt limits to Dev would be a good idea

**Comment by itinerare** [comment]
Yeah, I wouldn't be terribly opposed. Would like to know if others have thoughts on the matter, though.

**Comment by SpeedyD** [comment]
I think it's a decent idea as well.. Did you get permission from JuniJwi?

**Comment by ScuffedNewt** [comment]
nope, didn't plan on asking until there was a consensus on its addition! if everyone is cool with it I'll shoot them a dm

**Comment by ScuffedNewt** [comment]
got the go-ahead from juni 👍

**Comment by perappu** [comment]
Completed in https://github.com/lk-arpg/lorekeeper/pull/1389
